plugins/source/github/resources/services/users/keys.go

```diff
@@ -0,0 +1,45 @@
+package users
+
+import (
+	"context"
+
+	"github.com/apache/arrow/go/v15/arrow"
+	"github.com/cloudquery/cloudquery/plugins/source/github/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+	"github.com/google/go-github/v59/github"
+)
+
+func keys() *schema.Table {
+	return &schema.Table{
+		Name:      "github_user_keys",
+		Resolver:  fetchMembers,
+		Transform: client.TransformWithStruct(&github.Key{}, transformers.WithPrimaryKeys("ID")),
```

Comment:
Can we add a description and link to https://docs.github.com/en/rest/users/keys?apiVersion=2022-11-28#list-public-ssh-keys-for-the-authenticated-user?